File: src/SurveySays/wwwroot/ngApp/models/survey.ts
namespace SurveySays.Models {

    export interface ISurvey {
        id: number;
        surveyName: string;
        userId: string;
        dateCreated: any; //datetime
        lastModified: any; //datetime
        numResponses: number;
        numUnseenResponses: number;
        numQuestions: number;
        released: boolean;
        currentTroop: number;
        course: ICourse;
    }

    export class Survey implements ISurvey {
        id: number;
        surveyName: string;
        userId: string;
        dateCreated: any; //datetime
        lastModified: any; //datetime
        numResponses: number;
        numUnseenResponses: number;
        numQuestions: number;
        released: boolean;
        currentTroop: number;
        course: ICourse;
    }

}

File: src/Domain/Models/Survey.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Domain.Models
{
    public class Survey
    {
        public Survey()
        {
            QuestionSurveys = new List<QuestionSurvey>();
        }
        public int Id { get; set; }
        public string SurveyName { get; set; }
        public string UserId { get; set; }
        public Course Course { get; set; }
        public DateTime DateCreated { get; set; }
        public DateTime LastModified { get; set; }
        public int NumResponses { get; set; }
        public int NumUnseenResponses { get; set; }
        public int NumQuestions { get; set; }
        public bool Released { get; set; }
        public int CurrentTroop { get; set; }
        public bool IsActive { get; set; }

        public ICollection<QuestionSurvey> QuestionSurveys { get; set; }
    }
}
